src/d3d_prefida.pro

```diff
@@ -32,27 +34,27 @@ PRO d3d_prefida, inputs, igrid=igrid,bgrid=bgrid
         for i=0,n_elements(c)-1 do begin
             spec = d3d_chords(c[i], shot=inputs.shot)
             inputs.calc_fida = 1
-            inputs.calc_bes = 1
+            inputs.calc_nbi = 1
```

Comment:
You also need to turn on the `calc_halo`,`calc_dcx`, and `calc_fida` for now just leave the `calc_cold` and `calc_pfida` turned off.